pkg/asset/installconfig/powervs/client.go

```diff
@@ -390,6 +421,20 @@ func (c *Client) GetDNSInstancePermittedNetworks(ctx context.Context, dnsID stri
 	return networks, nil
 }
 
+// CreateDNSCustomResolver creates a custom resolver for the VPC.
+func (c *Client) CreateDNSCustomResolver(ctx context.Context, dnsID string, dnsZone string) (*dnssvcsv1.CustomResolver, error) {
+	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
```

Comment:
Do not discard the child context returned.
```suggestion
	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
```

Reply:
Actually removing this whole function as it's not needed currently.